app/src/main/java/com/example/giftly/AddEventScreen.java

```diff
@@ -109,7 +109,7 @@ public void onClick(View view) {
                             @Override
                             public void onSuccess(String eventID) {
                                 Log.d(TAG, "Successfully Created Event");
-                                Intent intent = new Intent(AddEventScreen.this, DisplayEventScreen.class);
```

Comment:
Possibly revise the code to ensure that the correct Intent is used for each case